components/ActionSheet/src/ColorThemer/MDCActionSheetColorThemer.h

```diff
@@ -0,0 +1,34 @@
+// Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
```

Comment:
`/**` notation for license preambles here and throughout.

Reply:
Done

Reply:
That was a typo. Please just match the existing style used throughout the code base.

Reply:
@jverkoey Do you want to revert #4478?

Reply:
We may need to look at that again. Why was /* not supported?

Reply:
Added a comment to that PR to provide more detail about the problems with `/* */` in file headers for Xcode. Is there an actual style guide requirement for license headers to only use `/* */` comments?

Reply:
Nope this just comes down to consistency.